GitCommands/Settings/AppSettings.cs

```diff
@@ -1512,6 +1512,8 @@ public static bool FillRefLabels
 
         public static ISetting<bool> MergeGraphLanesHavingCommonParent { get; } = Setting.Create(RevisionGraphSettingsPath, nameof(MergeGraphLanesHavingCommonParent), true);
 
+        public static ISetting<int> StraightenGraphSegmentsLimit { get; } = Setting.Create(RevisionGraphSettingsPath, nameof(StraightenGraphSegmentsLimit), 80);
```

Comment:
80 seem to be a lot.
Is this improving 'enough'?

Reply:
I did not see a difference between 80 and the former 40 (`MaxLanes`) with the Linux repo - based on #11456, (which is a draft yet because it has an issue with `merged graph lanes` when scrolling up. I hope I can work around this when I have improved the graph pipeline.)